students/zelichowski_michal/lesson_01_python_basics/fractional_part.py

```diff
@@ -0,0 +1,4 @@
+# Given a positive real number, print its fractional part.
+
+a = float(input())
+print(a - int(a))
```

Comment:
To jest ok, ale można też `a % 1` - zmierzyłem, że u mnie na komputerze jest 14x szybciej.
Popraw albo zostaw tak jak jest.